### DISCUSSION TITLE
Which companies are using rocket for the production?

### DISCUSSION THREAD

**Discussion by BadarSaghir** [rwf2/Rocket#2565] [General]
I want to learn rocket but I want to know which major companies are invested in this project? The only reason i ask because i really want to learn this framework but just want to know are their any companies taking this project seriously and giving the values to it what it deserves?

**Comment by kragil** [comment]
Probably not many anymore, because development has stalled to nearly total halt for years now.

**Reply by SergioBenitez** [reply]
> Probably not many anymore, because development has stalled to nearly total halt for years now.

There have been hundreds of commits in the last several months alone, including a new release candidate.

**Reply by kragil** [reply]
OK, that is true, but 0.5 RC history is: RC1 2021, RC2 2022, RC3 2023 ... 
Maybe get more people involved again and open it up, maybe then this project might have a brighter future and Rocket would be up-to-date again: https://github.com/flosse/rust-web-framework-comparison
